src/Map.js

```diff
@@ -1261,7 +1281,10 @@ export default class Map {
 
       // load google map
       await this._loadGoogleSatellite()
-
+      const mapCanvas = document.getElementById('map-canvas')
```

Comment:
Why don't you just simply put this in the `mount` method?
And again, don't use any document.getElementId with id name in the core, it is totally agnostic

Reply:
It was already in mount.
document.getElementId fixed